addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/receipt_header/receipt_header.js

```diff
@@ -13,4 +14,7 @@ export class ReceiptHeader extends Component {
             },
         },
     };
+    getServedByText() {
+        return _t("Served by %s", this.props.data.cashier);
+    }
```

Comment:
can you just change the text passed in the prop here?
https://github.com/odoo/odoo/blob/c18c9287c1ad54326119bdfd5c79ba18c89cc7d0/addons/point_of_sale/static/src/app/models/pos_order.js#L110

This way you don't need an extra method and an override and you won't even have to change anything in self order.

Reply:
I'll try that.

Reply:
It's unused in the order_receipt.xml.